roborazzi-compose-preview-scanner-support/src/main/java/com/github/takahirom/roborazzi/RoborazziPreviewScannerSupport.kt

```diff
@@ -13,49 +13,52 @@ import sergio.sastre.composable.preview.scanner.core.preview.ComposablePreview
 fun ComposablePreview<AndroidPreviewInfo>.captureRoboImage(
   filePath: String,
   roborazziOptions: RoborazziOptions = provideRoborazziContext().options,
-  configBuilder: RoborazziComposeConfigBuilder = this.toRoborazziComposeConfigBuilder()
+  roborazziComposeConfig: RoborazziComposeConfig = this.toRoborazziComposeConfig()
```

Comment:
@sergio-sastre 
I was almost ready to release this, but while writing the release notes, I realized it might be a bit strange to pass a builder to something. Additionally, the names `RoborazziComposeConfig` and `RoborazziComposeConfigBuilder` felt odd because `RoborazziComposeConfig` is not actually created by `RoborazziComposeConfigBuilder`. So I made some adjustments. Could you take a look?

Reply:
Sorry, I need to make some improvements. Please wait a moment. 🙇‍♂️